CHANGELOG.md

```diff
@@ -9,6 +9,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- Added a GFBio Collection Data Provider that replaces the custom GFBio API for accessing search results.
+
+  - https://github.com/geo-engine/geoengine/pull/697
+
 - Added a handler (`/layers/{provider}/{layer}/workflowId`) to register a layer by id and return the resulting workflow id.
 
```

Comment:
Is the coverage count from the recent build?
If it is and only about 40%, lets elaborate on which paths should be tested to get a decent amount.
If not, let's try to get a valid count.

Reply:
the bot comment is from before I added most of the tests and doesn't seem to be edited. Can we run it again somehow?

Reply:
Can we run this manually? :thinking: For the stats, we need coveralls, right?

Reply:
idk how :(

Reply:
I added a test that checks the meta data. Running tarpaulin locally outputs:

> 84.49% coverage, 43466/51446 lines covered